src/Affects/Connection/Events/Identified.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the tenancy/tenancy package.
+ *
+ * Copyright Laravel Tenancy & Daniël Klabbers <daniel@klabbers.email>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @see https://tenancy.dev
+ * @see https://github.com/tenancy
+ */
+
+namespace Tenancy\Affects\Connection\Events;
+
+use Tenancy\Affects\Connection\Contracts\ProvidesConfiguration;
+use Tenancy\Identification\Contracts\Tenant;
+
+class Identified
+{
+    /**
+     * @var Tenant|null
+     */
+    public $tenant;
+
+    /**
+     * @var ProvidesConfiguration
+     */
+    public $provider;
+
+    /**
+     * @var string|null
+     */
+    public $connection;
+
+    public function __construct(Tenant $tenant = null, string $connection = null, ProvidesConfiguration &$provider)
+    {
+        $this->tenant = $tenant;
+        $this->provider = &$provider;
```

Comment:
sorting is wrong, the $provider is the third argument, so it should be assigned third in the method as property